content_filter.py

```
@@ -1337,7 +1355,7 @@ async def notify_user_display_name(member, censored_display_name, server_config)
     except discord.Forbidden:
         pass
 
-async def notify_user_scan_deletion(message: discord.Message, censored_message: str, server_config: dict):
+async def notify_user_scan_deletion(message: discord.Message, censored_message: str, server_config: dict, removed_attachments: List[discord.Attachment] = [], blocked_exts: set = set()):
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)